README.md

```diff
@@ -63,11 +63,10 @@ For more information on the importance of a professional README for your reposit
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
+# 📖 [your_project_name] <a name="Awesome books: refactor to use JavaScript classes"></a>
```

Comment:
- [ ] kindly ensure your project has a title corresponding with what the project does, also remove the second line in project description , below is how your about section looks 
![image](https://github.com/Obote/AwesomeBooks/assets/61408860/cd91fa98-0acf-4d3e-a898-22cb892636ba)